modules/@apostrophecms/schema/index.js

```diff
@@ -251,7 +251,7 @@ module.exports = {
     });
 
     self.addFieldType({
-      name: 'checkboxes',
+      name: 'checkbox',
```

Comment:
This field literally contains an array of checkboxes, not just one, it should stay named `checkboxes`. If it were named `checkbox` I would expect something different.

If that means the component on the frontend has to be `checkboxes` too, then it should be. I realize this may generate discussion cc @abea @stuartromanek